### DISCUSSION TITLE
I made a Opencollective sponsorship couple of weeks ago, Need an update about it

### DISCUSSION THREAD

**Discussion by gitdevin99** [json-schema-org/community#800] [Q&A]
I submitted a monthly sponsorship request through OpenCollective last month and sent my website logo and URL via OpenCollective messages for inclusion on the home page. However, I have not received any response or seen my website URL added yet. Can someone please reply to me?

I also emailed benjamin.granados@postman.com and ben.hutton@postman.com, but both emails were returned.

How can I resolve this issue?

![Uploading Copycopter.ai - Open Collective.png…]()

**Comment by benjagm** [comment]
Hi @gitdevin99 ! Benjamin here. We have lost access to the Postman accounts this is why there was no reply. Next time you can contact the community by using info@json-schema.org

Can you please share a SVG version of your logo and a white one? We'll make an announcement ASAP.

**Comment by gitdevin99** [comment]
No worries, here is the SVG file

**Comment by benjagm** [comment]
Can you please send the SVG to the address info@json-schema.org?
